content/docs/guides/components/chainer.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Chainer Training"
+description = "Instructions for using Chainer for training"
+weight = 10
+toc = true
+bref= "This guide will walk you through using Chainer for training"
+[menu]
+[menu.docs]
+  parent = "components"
+  weight = 30
++++
+
+## Installing Chainer Operator
+
+If you haven't already done so please follow the [Getting Started Guide](/docs/started/getting-started/) to deploy Kubeflow.
+
+An **alpha** version of [Chainer](https://chainer.org/) support was introduced with Kubeflow 0.3.0. You must be using a version of Kubeflow newer than 0.3.0.
+
+## Verify that Chainer support is included in your Kubeflow deployment
+
+Check that the Chainer Job custom resource is installed
+
+```shell
+kubectl get crd
+```
+
+The output should include `chainerjobs.kubeflow.org`
+
+```
+NAME                                       AGE
+...
+chainerjobs.kubeflow.org                   4d
+...
+```
+
+If it is not included you can add it as follows
+
+```shells
+cd ${KSONNET_APP}
+ks pkg install kubeflow/chainer-job
+ks generate chainer-operator chainer-operator
+ks apply ${ENVIRONMENT} -c chainer-operator
+```
+
+## Creating an Chainer Job
+
+You can create an Chainer Job by defining an ChainerJob config file. See [examples/chainerjob-reference.yaml](https://github.com/kubeflow/chainer-operator/blob/master/examples/chainerjob-reference.yaml) config file. You may change the config file based on your requirements. By default, the example job is distributed learning with 3 nodes (1 master, 2 workers).
+
+```shell
+cat examples/chainerjob-reference.yaml
```

Comment:
This command doesn't seem very helpful because people won't have checked out the chainer repository.

I'd suggest just telling people to download/copy paste the link you provided above.

Reply:
You could also just past the YAML spec into the markdown doc and tell people to copy paste it.